### PR TITLE
Added CSS tag for 'footer a' element.

### LINKED ISSUE #9738: [Enhancement]On home page links in the footer area stand out
c3

### Please describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?

On the home page, the footer section shows different from other pages' footer which stands out in this sense.
Compare both for more details.

Also, divide the links section into two lists,

c2

> What did you expect to see that you didn't?

The result should be as below.
c2

### Please show us where to look

Compare https://publiclab.org/ with https://publiclab.org/wiki/welcome

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by cgrkzlkn**
Fixes #9738 
 
- As explained in this comment: https://github.com/publiclab/plots2/issues/9738#issuecomment-860958108 . I added these lines in [style.css](https://github.com/publiclab/plots2/blob/main/app/assets/stylesheets/style.css) instead of creating a new file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -238,16 +238,25 @@ To report security vulnerabilities or for questions about security, please conta
 Help improve Public Lab software!
 
 * Join the plots-dev@googlegroups.com discussion list to get involved
+* Most of the contributors network on our [gitter chat](https://gitter.im/publiclab/publiclab)
 * Look for open issues at https://github.com/publiclab/plots2/labels/help-wanted
 * We're specifically asking for help with issues labelled with [help-wanted](https://github.com/publiclab/plots2/labels/help-wanted) tag
 * Find lots of info on contributing at http://publiclab.org/wiki/developers
+<<<<<<< HEAD
```
Comment (afzal442) [inline README.md:245]: Plz, resolve the conflict!

**Review by afzal442 — APPROVED** [review]
looks good to me! Thanks.

File: app/views/dashboard/_node_meta.html.erb [hunk 2]
```diff
@@ -25,7 +25,11 @@
 </div>
 
 <div class="card-meta-links">
+<<<<<<< HEAD
```
Comment (afzal442) [inline app/views/dashboard/_node_meta.html.erb:28]: here too!

File: README.md (hunk 1 above)
Comment (cgrkzlkn) [reply README.md]: Hope it solved now, i deleted wrong commits

**Pull request closed without merging**